src/components/AnonymousChat/child/ChatReplyView.tsx

```diff
@@ -41,7 +41,9 @@ const ChatReplyView = ({type, messageType, replyData}: ChatReplyViewProps) => {
         )}
       </View>
 
-      <Text style={textStyle}>{replyData?.message}</Text>
+      <Text style={textStyle}>
+        {replyData?.attachments.length > 0 ? 'Photo' : replyData?.message}
+      </Text>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The current implementation assumes that if there are any attachments, they are always photos. This might not be accurate as the system also supports other types of attachments like videos. Consider checking the type of attachment before displaying the placeholder text.
```diff
- {replyData?.attachments.length > 0 ? 'Photo' : replyData?.message}
+ {replyData?.attachments.length > 0 ? getAttachmentType(replyData.attachments[0]) : replyData?.message}
```
In the above snippet, `getAttachmentType` is a hypothetical function that you would need to implement. It should return the type of the attachment (e.g., "Photo", "Video", etc.) based on the attachment data.


<!-- This is an auto-generated comment by OSS CodeRabbit -->